File: NRZ.Data/Aircraft.cs
using System;
using System.Collections.Generic;

namespace NRZ.Data
{
    public partial class Aircraft
    {
        public Aircraft()
        {
            AircraftDebt = new HashSet<AircraftDebt>();
            AircraftRegistration = new HashSet<AircraftRegistration>();
            AuctionItem = new HashSet<AuctionItem>();
            Distraint = new HashSet<Distraint>();
            RequestForCertificateOfDistraintOfProperty = new HashSet<RequestForCertificateOfDistraintOfProperty>();
        }

        public int Id { get; set; }
        public string ProducerName { get; set; }
        public string ProducerNameEn { get; set; }
        public string ProducerCountryCode { get; set; }
        public string ProducerCountryName { get; set; }
        public string AirCategoryCode { get; set; }
        public string AirCategoryName { get; set; }
        public string Icao { get; set; }
        public string MsnserialNumber { get; set; }
        public string ModelName { get; set; }
        public string ModelNameEn { get; set; }

        public virtual AircraftExtension AircraftExtension { get; set; }
        public virtual ICollection<AircraftDebt> AircraftDebt { get; set; }
        public virtual ICollection<AircraftRegistration> AircraftRegistration { get; set; }
        public virtual ICollection<AuctionItem> AuctionItem { get; set; }
        public virtual ICollection<Distraint> Distraint { get; set; }
        public virtual ICollection<RequestForCertificateOfDistraintOfProperty> RequestForCertificateOfDistraintOfProperty { get; set; }
    }
}

File: NRZ.Web/client/src/models/seizedAvailabiityReport/aircraft.ts
export default class Aircraft {
    producerName: string;
    producerNameEn: string;
    producerCountryCode: string;
    producerCountryName: string;
    airCategoryCode: string;
    airCategoryName: string;
    icao: string;
    msnSerialNumber: string;
    modelName: string;
    modelNameEn: string;

    constructor(obj: any = {}) {
        this.producerName = obj.producerName || '';
        this.producerNameEn = obj.producerNameEn || '';
        this.producerCountryCode = obj.producerCountryCode || '';
        this.producerCountryName = obj.producerCountryName || '';
        this.airCategoryCode = obj.airCategoryCode || '';
        this.airCategoryName = obj.airCategoryName || '';
        this.icao = obj.icao || '';
        this.msnSerialNumber = obj.msnSerialNumber || '';
        this.modelName = obj.modelName || '';
        this.modelNameEn = obj.modelNameEn || '';
    }
}
